### PR TITLE
Feature/add ourworks endpoint

### PR DESCRIPTION
**Pull request by C0Rs1K**

### CODE REVIEW

File: Curtains/Curtains.Domain/Models/BaseEntity.cs [hunk 1]
```diff
@@ -16,12 +16,8 @@ public class BaseEntity
         /// </summary>
         public int Id { get; set; }
         /// <summary>
-        ///  This property represents title of entities.
-        /// </summary>
-        public string Title { get; set; } = "";
```
Comment (zd-prog) [inline Curtains/Curtains.Domain/Models/BaseEntity.cs:21]: Why did you remove Title property?

Comment (C0Rs1K) [reply Curtains/Curtains.Domain/Models/BaseEntity.cs:21]: cuz it is not used in all models and conflicts with the structure of database

Comment (zd-prog) [reply Curtains/Curtains.Domain/Models/BaseEntity.cs:21]: Egor said to add Title and Description to BaseEntity class in previous pull request

Comment (C0Rs1K) [reply Curtains/Curtains.Domain/Models/BaseEntity.cs:21]: fixed :)

**Review by zd-prog — APPROVED** [review]

**Review by EgorAntonovich — APPROVED** [review]
